codeforamerica/vita-min
# NJ 124 - add Schedule NJ HCC xml and pdf when user indicates they had health insurance

## Link to pivotal/JIRA issue
https://github.com/newjersey/affordability-pm/issues/124

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

## What was done?
- Add Schedule NJ HCC to XML and PDF, if user indicated they had health insurance
- Fill name & SSN in PDF, check box for health coverage
- Do not add if user did not have health insurance

## How to test?
- to see XML and PDF, use any persona and select "yes" to health insurance question
- to see it not included, use an exception persona like `mfj_both_claimed_dep` and select "no" to health insurance question

## Screenshots (for visual changes)
![image](https://github.com/user-attachments/assets/eeed3bfd-67f3-4c40-97e8-8c364d2a92a5)
![image](https://github.com/user-attachments/assets/36a8e700-54f2-4707-a56f-383d15c418a3)
